media_driver/agnostic/gen10/vp/kernel/igvpdummykrn_g10.c

```diff
@@ -0,0 +1,32 @@
+/*                                                                                     
+ * Copyright (c) 2017, Intel Corporation                                               
+ *                                                                                     
+ * Permission is hereby granted, free of charge, to any person obtaining a             
+ * copy of this software and associated documentation files (the                       
+ * 'Software'), to deal in the Software without restriction, including                 
+ * without limitation the rights to use, copy, modify, merge, publish,                 
+ * distribute, sublicense, and/or sell copies of the Software, and to                  
+ * permit persons to whom the Software is furnished to do so, subject to               
+ * the following conditions:                                                           
+ *                                                                                     
+ * The above copyright notice and this permission notice shall be included             
+ * in all copies or substantial portions of the Software.                              
+ *                                                                                     
+ * THE SOFTWARE IS PROVIDED 'AS IS', WITHOUT WARRANTY OF ANY KIND, EXPRESS             
+ * OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF                          
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.              
+ * IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY                
+ * CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,                
+ * TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE                   
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                              
+*/                                                                                     
+                                                                                       
+////////////////////////////////////////////////////////////////////////////////       
+// !!! WARNING - AUTO GENERATED FILE. DO NOT EDIT DIRECTLY. !!!                        
+// Generated by KernelBinToSource.exe tool                                             
+////////////////////////////////////////////////////////////////////////////////       
+
+extern const unsigned int IGVPKRN_G10_SIZE = 216;
+extern const unsigned int IGVPKRN_G10[] = {
+    0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000,
```

Comment:
you define codec kernels with 0 size and VP kernels with non-zero. Why?

And the second question: you _define_ kernels even though they are trivial. Which behavior we should expect? Such kernel will fail to load? or we will get runtime error? /I will be able to check myself later/